scripts/geoJSON.js

```diff
@@ -0,0 +1,49 @@
+var config = require('./config.js')
+var coords = require('./coords.js')
```

Comment:
If you do the idea I propose in `index.js`, the `requires` above would be moved to `index.js` and the values would be passed into the `createGeoJSON()` below.